src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -20,6 +20,7 @@
  */
 public class Transaction {
   private static final int BASE_FEE = 100;
+  private static Integer defaultOperationFee;
```

Comment:
Can't we avoid the null checks and initialize this value:
```
private static int defaultOperationFee = BASE_FEE;
```
Otherwise isn't this a breaking change forcing code to change to specify the operation fee.